libraries/radspreadprocessing/features/comments.md

```diff
@@ -0,0 +1,189 @@
+
+***Properties***:
+
+* __CellIndex:__  Gets or sets the cell index assigned to the comment. 
+* __Text:__ Gets or sets the text of the comment. 
+* __CreationDate:__ Gets or sets the date when the comment is created. Nullable.
+* __Author:__ Gets or sets the author assigned to the comment.
+* __Replies:__ Gets the comment replies. The list is sorted by CreationDate.
+* __IsResolved:__ Gets or sets a value indicating whether the comment is resolved.
+
+ >note All of the above properties will push a change to the undo stack when modified. 
+ 
+***Methods***:
+
+* __AddReply:__ Adds a SpreadsheetCommentReply to the ReplySortedCollection.The collection will be re-sorted by SpreadsheetReply`s CreationDate in ascending order after adding an object.
+* __RemoveReply:__ Removes the specified reply from the collection.
+
+## Working with the CommentsCollection
+
+### Adding comments
+
+To add a comment you need to specify the cell index to which the comment will be related, the author, the text content, and the creation date. Specifying the creation date is optional and by default, its value is the current date and time.
+
+#### __[C#] Example 1: Add comment__
+
+{{region cs-radspreadprocessing-features-comments_0}}
+	
+    CellIndex relatedCellIndex = new CellIndex(1, 1);
+    string author = "John Doe";
+    string text = "Comment Content";
+	DateTime creationDate = DateTime.Now;
+
+	worksheet.Comments.Add(relatedCellIndex, author, text, creationDate);
+
+{{endregion}}
+
+The above snippet will add a comment in cell B2.
+
+
+### Removing Comments
+
+To remove a comment specify the comment instance, which can be obtained from the CommentsCollection. 
+
+
+#### __[C#] Example 2: Remove comment__
+
+{{region cs-radspreadprocessing-features-comments_1}}
+	
+    SpreadsheetComment comment = worksheet.Comments[0];
+	worksheet.Comments.Remove(comment);
+
+{{endregion}}
+
+The above snippet removes the first comment in the collection.
+
+## Replies
+
+Each comment can be replied to, forming a thread of information. All replies can be found in the __Replies__ property of the comment, which is of the type __ReplySortedCollection__. This collection holds __SpreadsheetCommentReply__ objects which represent the replies. The __ReplySortedCollection__ has the following members:
+
+***Properties***:
+
+* __Count:__ Gets the number of elements contained in the ReplySortedCollection.
+
+***Methods***:
+
+* __Add:__ Adds a __SpreadsheetCommentReply__ to the __ReplySortedCollection__. The collection will be re-sorted by __SpreadsheetReply`s__ CreationDate in ascending order after adding an object. Requires an object of type __SpreadsheetCommentReply__ and can be used for adding existing replies. For adding a new reply, it is best to use the __SpreadsheetComment.AddReply()__ method.
+* __Remove:__ Removes the first occurrence of a specific object from the __ReplySortedCollection__.
```

Comment:
I don't think we could have the same object added twice to the collection. Let's change that to "Removes the specified SpreadsheetCommentReply object from the __ReplySortedCollection__."